### PR TITLE
updating current drought layer 20150714

### PR DESCRIPTION
**Pull request by eread-usgs**

### CODE REVIEW

**Comment by eread-usgs** [comment]
@jread-usgs @lawinslow updated the viz and it all seemed fine, no errors, but this doesnt look quite right: http://rawgit.com/USGS-CIDA/CIDA-Viz/master/Vizzies/public_html/index.html

going to wait to hear from you two before doing the jenkins job

**Comment by eread-usgs** [comment]
@jread-usgs @lawinslow ok, i think the script is actually just skipping 20150714 for some reason
not sure what to do about that.....

**Comment by jiwalker-usgs** [comment]
Looks like the problem is main.js timesteps has invalid quote character ˵ ˶ instead of "

**Comment by eread-usgs** [comment]
uh oh
my bad!!!!!!!!!!!!!!!

File: Vizzies/public_html/js/main.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
..."20121127", "20121120", "20121113", "20121106", "20121030", "20121023", "20121016", "20121009", "20121002", "20120925", "20120918", "20120911", "20120904", "20120828", "20120821", "20120814", "20120807", "20120731", "20120724", "20120717", "20120710", "20120703", "20120626", "20120619", "20120612", "20120605", "20120529", "20120522", "20120515", "20120508", "20120501", "20120424", "20120417", "20120410", "20120403", "20120327", "20120320", "20120313", "20120306", "20120228", "20120221", "20120214", "20120207", "20120131", "20120124", "20120117", "20120110", "20120103", "20111227", "20111220", "20111213", "20111206", "20111129", "20111122", "20111115", "20111108", "20111101", "20111025", "20111018", "20111011", "20111004", "20110927", "20110920", "20110913", "20110906", "20110830", "20110823", "20110816", "20110809", "20110802", "20110726", "20110719", "20110712", "20110705", "20110628", "20110621", "20110614", "20110607", "20110531", "20110524", "20110517", "20110510", "20110503", "20110426", "20110419", "20110412", "20110405", "20110329", "20110322", "20110315", "20110308", "20110301", "20110222", "20110215", "20110208", "20110201", "20110125", "20110118", "20110111", "20110104"];
```
Comment (jiwalker-usgs) [inline Vizzies/public_html/js/main.js]: right here “ is not valid in javascript

**Comment by eread-usgs** [comment]
i used the wrong text editor

**Comment by eread-usgs** [comment]
I will fix it

**Pull request merged**